src/olympia/migrations/929-add-user-auth-id.sql

```diff
@@ -0,0 +1,3 @@
+-- note: the column is added as nullable, a command will backfill values for
+-- existing users.
+ALTER TABLE `users` ADD COLUMN `auth_id` integer UNSIGNED;
```

Comment:
I thought about that, and making it a `VARCHAR` with more entropy, but I don't think it's necessary. It will only be used as a component of the token (which will depend on a timestamp, user id and our secret key as well) to invalidate user sessions, so it doesn't matter that much that it's not unique across different users.